src/main/python/fever/convert_collection_to_jsonl.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
```

Comment:
We've tweaked our header, please use this? https://github.com/castorini/anserini-tools/blob/master/scripts/msmarco/convert_msmarco_to_trec_qrels.py#L1

Also, we recently learned that according to PEP8, the files _shouldn't_ have coding directive.

Reply:
Updated!